bundle/src/main/java/com/adobe/acs/commons/indesign/dynamicdeckdynamo/osgiconfigurations/DynamicDeckConfigurationService.java

```diff
@@ -0,0 +1,36 @@
+package com.adobe.acs.commons.indesign.dynamicdeckdynamo.osgiconfigurations;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.metatype.annotations.Designate;
+
+@Component(immediate = true, service = DynamicDeckConfigurationService.class)
```

Comment:
Any reason why this is immediate?
Also, any reason why you arent splitting the service by interface/impl - typically this sepereation is preferred (as its more flexible) and you seemed to have applied it elsewhere in the code base.